packages/models/test/fixtures/eslint-audits.mock.ts

```diff
@@ -838,4 +838,4 @@ export const ESLINT_AUDITS_MAP = {
     docsUrl:
       'https://github.com/jsx-eslint/eslint-plugin-react/tree/master/docs/rules/require-render-return.md',
   },
-} satisfies Record<string, AuditReport>;
+};
```

Comment:
Please revert this :pray: It removes the type-safety of the audit slugs, I'm using `keyof typeof ESLINT_AUDITS_MAP` (I want that to be a union, not a `string`) to make the audit references more manageable :sweat_smile:

Reply:
reverted